resolwe_bio/docker_images/rnaseq/packages-python3.txt

```diff
@@ -1,5 +1,5 @@
-cutadapt==1.14 --hash=sha256:9c1423123e764bc95e96d196ee8025a5ec213ffab218fc09088bdd2b577b3a04
-HTSeq==0.9.1 --hash=sha256:f61cd2a19aa12514b3b9d4fa0725cbf8d34b06313da6c468d38cfe9aef84b350
-pyarrow==0.7.1 --hash=sha256:7908c71b0109d6f622436bdf719cde5b749287a697134d23040e35d32c28a313
-pysam==0.12.0.1 --hash=sha256:a99cd626ff0fdc09b68a1cc9cc9551f4dc3c4cd49872583de1a2f39e69520923
+cutadapt==1.14 --hash=sha256:a9903b9288956c6e8c4652670efa4eda40b258f29f00f3b85ae6b16d80b022c8
```

Comment:
While you're at it, check the latest version of all python packages installed in the RNA-seq docker image. All packages except HTSeq can be updated. This should become our standard practice to keep the Docker images well maintained.